includes/admin/settings/class.llms.settings.page.php

```diff
@@ -78,59 +108,107 @@ public function flush_rewrite_rules() {
 	 * @since 3.17.5
 	 * @since 3.35.0 Unslash input data.
 	 *
-	 * @return   string
+	 * @return string
 	 */
 	protected function get_current_section() {
-
 		return isset( $_GET['section'] ) ? sanitize_text_field( wp_unslash( $_GET['section'] ) ) : 'main';
+	}
+
+	/**
+	 * Retrieve the page label.
+	 *
+	 * Extending classes should override this to return a translated string used as the page's title.
+	 *
+	 * @since [version]
+	 *
+	 * @return string
+	 */
+	public function get_label() {
+		return $this->id;
+	}
+
+	/**
+	 * Generates a group of settings.
+	 *
+	 * @since [version]
+	 *
+	 * @param string  $id Group ID. Used to create IDs for the start, end, and title fields.
+	 * @param string  $title Title of the group (should be translatable).
+	 * @param string  $title_desc (Optional) title field description text.
```

Comment:
If we follow the wp doc standards, as I think we do (?), we should have:
```
* @param string  $title_desc Optional. Title field description text. Default empty.
* @param array[] $settings   Optional. Array of settings field arrays. Default empty array.
```

see: https://github.com/WordPress/WordPress/blob/5.3/wp-includes/rest-api/class-wp-rest-request.php#L108

Reply:
This hasn't actually ever been explicitly defined in the coding standards and I haven't been doing it anymore since I rewrote them. Why? Because I haven't been able to find any way to automate this through PHPCBF or any other tools.

The WordPress core does not specify any amount of alignment either. If you have a look through the core codebase *sometimes* doc blocks are aligned and sometimes they're not. The same is true of LifterLMS.

I think they do *look* a whole lot nicer when aligned however without a way to automatically identify (and throw errors) as well as automatically fix it it feels like a large amount of work to enforce with little benefit as the alternative is still pretty easy to read as a human and the phpdoc parser that builds our developer.lifterlms.com site doesn't need any special alignment to parse.

If you can find a way to sniff and fix this I'd very much like to enforce this alignment but until then I'm not going to worry about it.

Reply:
@thomasplevy I see your point, thanks.